Mathlib/Analysis/SpecialFunctions/Integrals.lean

```diff
@@ -741,6 +741,19 @@ theorem integral_cos_sq : ∫ x in a..b, cos x ^ 2 = (cos b * sin b - cos a * si
   by field_simp [integral_cos_pow, add_sub_assoc]
 #align integral_cos_sq integral_cos_sq
 
+theorem integral_sqrt_one_sub_sq : ∫ x in (-1 : ℝ)..1, sqrt (↑1 - x ^ 2) = π / 2 :=
```

Comment:
It is quite surprising to me that when the `↑` is present, there's no coercion `ℕ → ℝ`, but there is if you remove the `↑`. I would use `(1 : ℝ)` for clarity, though

Reply:
I put `↑` because that was what Lean suggested. However, you are right that putting `(1 : ℝ)` in the statement would make things more clear.